runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -4053,22 +4138,67 @@ MM_CopyForwardScheme::clearCardTableForPartialCollect(MM_EnvironmentVLHGC *env)
 }
 
 UDATA
-MM_CopyForwardScheme::alignMemoryPool(MM_EnvironmentVLHGC *env, MM_MemoryPoolBumpPointer *pool)
+MM_CopyForwardScheme::alignMemoryPool4Collector(MM_EnvironmentVLHGC *env, MM_MemoryPoolAddressOrderedList *regionPool)
```

Comment:
This alignment pass can now be significantly more expensive than before. Since this is done under region list lock, this may impede efficiency of other GC threads. I'd like to do it lazily as we do collector(TLH)Allocate (hence spread the cost among threads).
We can discuss details, but
- one possibility (and likely preferred) is to have separate API called before collector(TLH)Allocate, and pool (region) local address that maintains progress up to which point alignment is done 
- or we could extend existing MPAOL collectorTLHAllocate APIs with alignment parameter (in standard collectors, the alignment would be regular heap alignment of 8 bytes)